cost/azure/instances_log_analitics_utilization/azure_instance_log_analytics_utilization.pt

```diff
@@ -0,0 +1,521 @@
+name "Azure Instances Utilization from Log Analytics"
+rs_pt_ver 20180301
+type "policy"
+short_description "Gathers instance utilization data from Azure Log Analytics and tags underutilized instances.  See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/azure/instances_log_analitics_utilization) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+severity "low"
+category "Cost"
+
+parameter "param_azure_tenant_id" do
+  type "string"
+  label "Azure AD Tenant ID"
+  category "Azure"
+end
+
+parameter "param_azure_sub" do
+  type "string"
+  label "Azure Subscription ID"
+  category "Azure"
+end
+
+parameter "param_avg_used_memory" do
+  type "number"
+  label "Average used memory percent to allow for downsize"
+  default -1
+  min_value -1
+  max_value 100
+end
+
+parameter "param_avg_cpu" do
+  type "number"
+  label "Average cpu percent to allow for downsize"
+  default -1
+  min_value -1
+  max_value 100
+end
+
+parameter "param_exclusion_tag_key" do
+  category "User Inputs"
+  label "Exclusion Tag Key"
+  description "Cloud native instance tag to ignore instances"
+  type "string"
+  default "exclude_downsize"
+end
+
+parameter "param_take_action_tag" do
+  category "User Inputs"
+  label "Downsize Tag Key:Value"
+  description "Cloud native instance tag name and value that will be added to an instance identified for action"
+  type "string"
+  default "downsize:true"
+end
+
+parameter "param_email" do
```

Comment:
Need to shorten the label and add description.  
```
parameter "param_email" do
  type "list"
  label "Email addresses"
  description "Email addresses of the recipients you wish to notify"
end```

Reply:
Done.